zerver/lib/test_helpers.py

```diff
@@ -808,3 +816,18 @@ def dns_txt_answer(name_str: str, txt: str) -> dns.resolver.Answer:
         dns.rdtypes.ANY.TXT.TXT(rdclass, rdtype, txt)
     )
     return dns.resolver.Answer(name, rdtype, rdclass, response)
+
+
+def build_streams_subscriber_count(streams: Iterable[Stream]) -> dict[int, int]:
+    """
+    Callers MUST pass a new db-fetched version of streams each time.
+    """
+    return {stream.id: stream.subscriber_count for stream in streams}
+
+
+def fetch_streams_subscriber_count(stream_ids: set[int]) -> dict[int, int]:
+    return build_streams_subscriber_count(streams=Stream.objects.filter(id__in=stream_ids))
+
+
+def fetch_other_streams_subscriber_count(stream_ids: set[int]) -> dict[int, int]:
+    return build_streams_subscriber_count(streams=Stream.objects.exclude(id__in=stream_ids))
```

Comment:
I think these might be better if defined on `ZulipTestCase` so you don't need to import the functions by name just to use these test helpers.

Reply:
done